test/lib/Benchmark.js

```diff
@@ -299,4 +309,15 @@ Benchmark.prototype = {
   })
 };
 
+const PREF_LOG_LEVEL = `extensions.${self.id}.sdk.console.logLevel`;
+
+// bLog allows the user to use the console.log() regardless the current log level
```

Comment:
It's not obvious to me what the b in bLog stands for.  Maybe call this forcedLog?